Makefile

```diff
@@ -38,10 +39,16 @@ $(DOCKER_DISTRIB):
 	curl -o $(DOCKER_DISTRIB) $(DOCKER_DISTRIB_URL)
 
 docker/weave:
-	curl -L git.io/weave -o docker/weave
+	curl -L https://github.com/weaveworks/weave/releases/download/v$(WEAVENET_VERSION)/weave -o docker/weave
 	chmod u+x docker/weave
 
-$(SCOPE_EXPORT): $(SCOPE_EXE) $(DOCKER_DISTRIB) docker/weave $(RUNSVINIT) docker/Dockerfile docker/demo.json docker/run-app docker/run-probe docker/entrypoint.sh
+docker/weaveutil:
+	docker pull weaveworks/weaveexec:$(WEAVENET_VERSION)
```

Comment:
Apparently I cannot do this because Docker 1.10.2 doesn't like running the weave-exec container. Making it fail in CircleCI

```
vagrant@vagrant-ubuntu-wily-64:~/scope$ docker run  weaveworks/weaveexec:1.9.0 true
docker: Cannot connect to the Docker daemon. Is the docker daemon running on this host?.
See 'docker run --help'.
vagrant@vagrant-ubuntu-wily-64:~/scope$ docker version
Client:
 Version:      1.10.2
 API version:  1.22
 Go version:   go1.5.3
 Git commit:   c3959b1
 Built:        Mon Feb 22 21:40:35 2016
 OS/Arch:      linux/amd64

Server:
 Version:      1.10.2
 API version:  1.22
 Go version:   go1.5.3
 Git commit:   c3959b1
 Built:        Mon Feb 22 21:40:35 2016
 OS/Arch:      linux/amd64
vagrant@vagrant-ubuntu-wily-64:~/scope$ docker run busybox echo hello
hello
```